maven-surefire-common/src/test/java/org/apache/maven/plugin/surefire/extensions/StreamFeederTest.java

```diff
@@ -90,9 +91,9 @@ public void shouldEncodeCommandToStream() throws Exception
                 public Object answer( InvocationOnMock invocation ) throws IOException
                 {
                     ByteBuffer bb = invocation.getArgument( 0 );
```

Comment:
Maybe it would be nicer to declare the type here and then we do not have to cast `bb`.

Reply:
I mean to declare the type as Buffer, see my proposal `Buffer bb = invocation.getArgument( 0 );`

Reply:
`bb.array()` two lines further down would then return `Object` instead of `byte[]` which would mean a cast in that place instead. Is that preferable in your opinion?